dts/bindings/gpio/nordic,nrf-gpio-forwarder.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This is an abstract device responsible for forwarding pins between cores.
+
+    Here is an example of how a nrf-gpio-forwarder can be used with a nRF5340
+    combined with a nRF21540 Front-End module. Consider the following node
+    present in DTS file targeted for the nRF5340 network core:
+
+    nrf_radio_fem: nrf21540 {
+      compatible = "nordic,nrf21540-fem";
+      tx-en-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+      rx-en-gpios = <&gpio1 11 GPIO_ACTIVE_HIGH>;
+      pdn-gpios   = <&gpio1 10 GPIO_ACTIVE_HIGH>;
+      mode-gpios  = <&gpio1 12 GPIO_ACTIVE_HIGH>;
+    };
+
+    and the following node defined in DTS file targeted for the nRF5340
+    application core:
+
+    gpio_fwd: nrf-gpio-forwarder {
+      compatible = "nordic,nrf-gpio-forwarder";
+      nrf21540-gpio-if {
+        gpios = <&gpio0 30 0>, <&gpio1 11 0>, <&gpio1 10 0>, <&gpio1 12 0>;
+      };
+    };
+
+    In the above example, the nrf-gpio-forwarder node is configured to forward
+    control over the following GPIO pins:
```

Comment:
Forward control from where to where? What happens when control is forwarded?

Reply:
(To clarify, I'm asking both for myself and because I think this information is missing from the description)

Reply:
I extended the documentation to address your comments, please check if contains all the answers now :)